unit_tests/UnitTestHexMasterElementsNgp.C

```diff
@@ -380,8 +384,8 @@ TEST_F(MasterElementHexSerialNGP, hex27_scs_interpolation)
 {
   if (stk::parallel_machine_size(comm) == 1) {
     setup_poly_order_2_hex_27();
-    sierra::nalu::Hex27SCS hex27scs;
-    //check_interpolation<sierra::nalu::AlgTraitsHex27>(meta, bulk, hex27scs);
+    //using AlgTraits = sierra::nalu::AlgTraitsHex27;
+    //check_interpolation<AlgTraits, AlgTraits::masterElementScs_, true>(meta, bulk);
```

Comment:
Is this commented out deliberately?

Reply:
Yes, I have not converted the Hex27 classes to GPU.  Hopefully they are just like the Hex8.